chart/Chart.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v2
+name: sparrow
+description: A Helm chart to install Sparrow
+type: application
+keywords:
+  - monitoring
+version: 0.0.2
+appVersion: "v0.0.0-SNAPSHOT-6477ffc"
```

Comment:
We'll have to change the `appVersion` to something else, this SNAPSHOT thing is a maven/graddle convention.

Reply:
Yes, just wondering there are no tagged versions so far.

Reply:
The SNAPSHOT builds are available for every push for testing purposes. As @eumel8 stated, as soon as we will have the first release, the chart should ref this released appVersion.

Reply:
Should we create a pre release?

Reply:
Fine by me, thanks for implementing this one :partying_face:

Reply:
new image tag will be `commit-<smallsha>`